pkg/providers/cloudstack/cloudstack_test.go

```diff
@@ -220,6 +220,32 @@ func TestProviderGenerateCAPISpecForCreate(t *testing.T) {
 	test.AssertContentToFile(t, string(md), "testdata/expected_results_main_md.yaml")
 }
 
+func TestProviderGenerateDeploymentFileWithNonEksaResources(t *testing.T) {
```

Comment:
Curious, what is this test for? It doesn't seem like the cloudstack code has changes

Reply:
Pushed.